### ISSUE TITLE
Attiny85

### ISSUE THREAD

**Issue by sniperxnl** [YannikW/IR-Laptimer#1]
Can you make a code that send only one ID and works on the Attiny85?

**Comment by YannikW** [comment]
Hey, sure - I will do ;) But can you tell me what pin the led/transisor is connected to?

**Comment by sniperxnl** [comment]
I use pin6

**Comment by YannikW** [comment]
Please try this software : https://github.com/YannikW/Open-Lap/tree/master/Transponder/Software/Fix%20ID%20on%20Attiny85%20or%20Attiny45

I tested it on an ATTiny85 with LED on Pin 6. Should work for u ;)

Make sure you first burn the bootloader to enable 8Mhz internal clock ;) Default I think it runs on the 1Mhz clock, then timing will be incorrect.

**Comment by Cwix** [comment]
I can't seem to get Attiny85 to work. I connected the esp board and flashed the Digispark board which uses the Attiny85. Connected everything but can't seem to get usable data.

When I enable DEBUGMODE 2 in the console i get a lot of incomplete data but nothing usable. Should modulated signal be visible when transmitter is sending out(by using visible diode or a camera to see IR)?

All I get is something like this:
1 NEB
1 NEB
1 NEB
1 NEB
1 NEB
1 NEB
1 NEB
1 NEB
1 NEB
1 IT 96
1 NEB
1 NEB
1 NEB
1 IT 73
1 IT 96
1 NEB
1 NEB
1 IT 96
1 NEB

**Comment by YannikW** [comment]
Hmm sounds like the Attiny runs in a wrong clock source or something like this. You changed it to 8Mhz (Internal) and burned bootloader after the change?

EDIT : And yes, with a camera IR Blinking should be visible! Care if you use the SFH4346.. The polarity of this diode in inversed.. Flat/shorter side is anode and long side is kathode.. Took some time at my first test to check this :P

**Comment by RaceFPV** [comment]
Attiny85's internal clock is notoriously crap, I'd recommend an external clock.

**Comment by YannikW** [comment]
Yeah, on original OpenLap Transponder there is an Attiny44 with external 16Mhz, but this Attiny85 Transponders are from an other project which don't uses external clock.. I also recommend using external one!

**Comment by Cwix** [comment]
Oh, I'll try to find the external oscillator ang give it a try. Thanks!
